### PR TITLE
1917 - from post list articles launch in main view. 

### PR DESCRIPTION
**Pull request by relativityboy**
#1917

### CODE REVIEW

**Comment by relativityboy** [comment]
@valzav would you take a close look at the updates to useScroll in `UniversalRender.jsx`?

You were the last one to touch it, but the old logic looked to be out of date.

File: src/shared/UniversalRender.jsx [hunk 1]
```diff
@@ -82,15 +82,16 @@ async function universalRender({ location, initial_state, offchain, ErrorPage, t
 
     if (process.env.BROWSER) {
         const store = createStore(rootReducer, initial_state, middleware);
+        window.store = store; //todo: remove
```
Comment (goldibex) [inline src/shared/UniversalRender.jsx]: what's up with this here?

Comment (relativityboy) [reply src/shared/UniversalRender.jsx]: Justifying the review...

**Comment by bnchdrff** [comment]
works for me

but... should we change all the other instances of `e => navigate`? the glaring one for me is at https://github.com/steemit/condenser/pull/1919/files#diff-8a794293a156dbd272160f7e96c0b313R224 -- clicking on the lead image for a post will still open it popup style

**Pull request merged**